getting-started/installing-node: add manual install instructions

williamboman opened the pull request.

ashleygwilliams commented:
given our release cycle, i think maintaining the specific tar will be too much. if you'd like to rewrite it so that it just point to the module and explains how to find the tar, i'd consider it. @williamboman

williamboman commented:
I remember having a hard time finding the tarball, hence the PR :). Not sure how I eventually found it, it wasn't from official sources though. I suppose `$ npm view npm dist.tarball` would be the easiest way, but that assumes you have `npm` installed already.

Would just changing the URL to `https://registry.npmjs.org/npm/-/npm-{VERSION}.tgz` do it?

Also, out of curiosity, are there any plans to use a static site generator for generating the docs?

ashleygwilliams commented:
hey @williamboman sorry for the delay in response!

i think that update to the URL is :+1: , if you make that change and rebase that'd be rad

re: static site generator... yes! i have a repo under my personal account where i am working on that :)
was thinking of using [metalsmith](http://www.metalsmith.io/). any thoughts?

williamboman commented:
Rebased and updated.

> re: static site generator... yes! i have a repo under my personal account where i am working on that :)
> was thinking of using metalsmith. any thoughts?

I've only used Jekyll before so I wouldn't really know tbh.

ashleygwilliams commented on content/getting-started/installing-node.md:
```diff
@@ -20,3 +20,9 @@ Node comes with npm installed so you should have a version of npm. However, npm
+
+> For more advanced users.
+
+The npm module is available for download at https://registry.npmjs.org/npm/-/npm-{VERSION}.tgz.
```
final nit: could you put the address as an inline code block a la `https://registry.npmjs.org/npm/-/npm-{VERSION}.tgz`

williamboman replied:
oh, though I already had done that. updated

ashleygwilliams commented:
ok one final nit and then i''ll merge! thanks so much for contributing! would you like some npm socks? contributing to docs entitles you to some :smile:

williamboman commented:
> would you like some npm socks? contributing to docs entitles you to some

Haha sure. Does this apply to non-US people too (shipping etc.)?

ashleygwilliams commented:
it does! :)

ashleygwilliams commented:
should be on prod in a minute!
